src/stylesheets/index.css

```diff
@@ -1,22 +1,21 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
These 3 declarations will import all of Tailwind's CSS. When we create a production build, all the unused CSS will be removed automatically.